app/src/main/java/com/belatrixsf/allstars/adapters/RankingListAdapter.java

```diff
@@ -0,0 +1,150 @@
+* FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+* AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+* LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+* OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+* SOFTWARE.
+*/
+package com.belatrixsf.allstars.adapters;
+
+import android.content.Context;
+import android.support.v7.widget.RecyclerView;
+import android.view.LayoutInflater;
+import android.view.View;
+import android.view.ViewGroup;
+import android.widget.ImageView;
+import android.widget.TextView;
+
+import com.belatrixsf.allstars.R;
+import com.belatrixsf.allstars.entities.Employee;
+import com.belatrixsf.allstars.ui.common.views.BorderedCircleTransformation;
+import com.belatrixsf.allstars.ui.common.views.CircleTransformation;
+import com.belatrixsf.allstars.utils.Constants;
+import com.bumptech.glide.Glide;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import butterknife.Bind;
+import butterknife.ButterKnife;
+import butterknife.OnClick;
+
+/**
+ * Created by icerrate on 28/04/2016.
+ */
+public class RankingListAdapter extends RecyclerView.Adapter<RankingListAdapter.EmployeeViewHolder> {
+
+    private List<Employee> rankingList;
+    private RankingListClickListener rankingListClickListener;
+
+    public RankingListAdapter(RankingListClickListener rankingListClickListener) {
+        this(rankingListClickListener, new ArrayList<Employee>());
+    }
+
+    public RankingListAdapter(RankingListClickListener rankingListClickListener, List<Employee> rankingList) {
+        this.rankingList = rankingList;
+        this.rankingListClickListener = rankingListClickListener;
+    }
+
+    @Override
+    public EmployeeViewHolder onCreateViewHolder(ViewGroup parent, int viewType) {
+        View layoutView = LayoutInflater.from(parent.getContext()).inflate(R.layout.item_ranking, parent, false);
+        return new EmployeeViewHolder(layoutView, rankingListClickListener);
+    }
+
+    @Override
+    public void onBindViewHolder(EmployeeViewHolder holder, int position) {
+        final Employee employee = rankingList.get(position);
+        int place = position + Constants.ONE_UNIT;
+        int cupResourceId;
+        if (place<=3) {
+            int crownResourceId;
+            switch (place) {
+                case 1:
+                    crownResourceId = R.drawable.ic_first_place;
+                    cupResourceId = R.drawable.ic_first_place_cup;
+                    break;
+                case 2:
+                    crownResourceId = R.drawable.ic_second_place;
+                    cupResourceId = R.drawable.ic_second_place_cup;
+                    break;
+                default:
+                    crownResourceId = R.drawable.ic_third_place;
+                    cupResourceId = R.drawable.ic_third_place_cup;
+                    break;
```

Comment:
I think this is not maintainable maybe we can create a static final array, something like:

public static final CROWN_RESOURCES[] = {..., ..., ...}
public static final CUP_RESOURCES[] = {..., ..., ...} 

and use crownResource = CROWN_RESOURCES[place-1] same for cups and put the constants in the Constants class